ironfish-cli/src/commands/service/faucet.ts

```diff
@@ -8,6 +8,7 @@ import { RemoteFlags } from '../../flags'
 
 const FAUCET_AMOUNT = 5
 const FAUCET_FEE = 1
+const MAX_RECIPIENTS_PER_TRANSACTION = 5
```

Comment:
If we have 5, it'll take 6 hours best case scenario, most likely double or triple this. If there any number want such a low number?

Reply:
This was a test value intended for discussion.